change/@fluentui-react-native-radio-group-2020-06-04-14-39-18-link-focus.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Any idea what happened with all these change jsons? I'm guessing you only need the @fluentui-react-native-link one.

Reply:
when I run "yarn change" it asks me if there are changes in all these packages. Even when I choose "none" for change type, these json files get created. Any way I could avoid that?